test/cljs/status_im/test/chat/models/input.cljs

```diff
@@ -3,7 +3,7 @@
             [status-im.chat.models.input :as input]))
 
 (def fake-db
-  {:global-commands   {:command1 {:name "global-command1"}}
+  {:global-commands   {:command1 '({:name "global-command1"})}
```

Comment:
Can you explain why we need this quoting? It's only tests but I don't understand why

Reply:
just to create a list i suppose. But maybe better to use `list` here

Reply:
If it is just changing collection type from map to list a vector seems more idiomatic. Is there something list-specific about this?

Reply:
Let's keep lists here because we use lists in the real db. I will replace `'(...)` with `(list ...)` to improve readability.